axmysql/messagestore/append.go

```diff
@@ -146,6 +146,12 @@ func insertMessage(
 		return err
 	}
 
+	// Truncate the message to 255 characters to fit in the schema restriction;
+	// see
+	// https://github.com/jmalloc/ax/blob/v0.4.0/axmysql/messagestore/schema.sql#L37
+	// for details.
```

Comment:
```suggestion
	// Truncate the message to 255 characters to fit within the column.
```

Just removing links to specific tags.